ovh/resource_domain_zone_record.go

```diff
@@ -269,6 +278,9 @@ func ovhDomainZoneRecord(client *ovh.Client, d *schema.ResourceData, id string,
 	})
 
 	if err != nil {
+		if errOvh, ok := err.(*ovh.APIError); ok && errOvh.Code == 404 {
```

Comment:
i think this is useless as the call to CheckDeleted right after is meant to check this 404.